src/interfacers/EmonHubBleInterfacer.py

```diff
@@ -0,0 +1,168 @@
+import time
+import struct
+from bluepy import btle
+import Cargo
+from emonhub_interfacer import EmonHubInterfacer
+
+"""class EmonhubBleInterfacer
+
+Polls a Bluetooth LE sensor for temperature, huimidity and battery level
+
+Currently only tested with a Silicon Labs Thunderboard Sense 2
+
+
+Example config snippets:
+
+[interfacers]
+
+[[blesensor]]
+    Type = EmonHubBleInterfacer
+    [[[init_settings]]]
+        device_addr = '00:0b:57:64:8c:a2'
+    [[[runtimesettings]]]
+        pubchannels = ToEmonCMS,
+	read_interval = 20
+
+[nodes]
+
+[[1]]
+    nodename = Sensornode
+
+    [[[rx]]]
+        names = temp,humidity,battery
+        scales = 0.01,0.01,1
+        units = C,%,%
+
+"""
+
+class EmonHubBleInterfacer(EmonHubInterfacer):
+
+    def __init__(self, name, device_addr=''):
```

Comment:
This isn't a useful default for device_addr, and it's an error if no addr is supplied, remove =''

Reply:
done